src/domain/wallets/index.types.d.ts

```diff
@@ -82,5 +82,5 @@ type Wallet = {
 interface IWalletsRepository {
   findById(walletId: WalletId): Promise<Wallet | RepositoryError>
   findByWalletName(walletName: WalletName): Promise<Wallet | RepositoryError>
-  findByAddresses(addresses: string[]): Promise<Wallet[] | RepositoryError>
+  listByAddresses(addresses: string[]): Promise<Wallet[] | RepositoryError>
```

Comment:
in a previous PR you told me that find prefix should be used when we have a search criteria, list prefix when we have no params